external/concealer/src/field_operator_application_for_autoware_universe.cpp

```diff
@@ -192,6 +206,34 @@ auto toModuleType(const std::string & module_name)
   }
 }
 
+template <typename CooperateStatusType>
+bool isValidCooperateStatus(
+  const CooperateStatusType & cooperate_status, std::uint8_t command_type, std::uint8_t module_type)
+{
+  /**
+   *  NOTE1: the finish_distance filter is set to over -20.0,
+   *  because some valid rtc statuses has negative finish_distance due to the errors of
+   * localization or numerical calculation. This threshold is advised by a member of TIER IV
+   * planning and control team.
```

Comment:
```suggestion
   * NOTE1: the finish_distance filter is set to over -20.0,
   * because some valid rtc statuses has negative finish_distance due to the errors of
   * localization or numerical calculation. This threshold is advised by a member of TIER IV
   * planning and control team.
```